test/bats.bats

```diff
@@ -191,6 +191,105 @@ fixtures bats
   [ "${lines[2]}" = "# (in test file $FIXTURE_ROOT/failing.bats, line 4)" ]
 }
 
+@test "find_library_load_path finds single-file libraries with the suffix .bash" {
+  lib_dir="$BATS_TMPNAME/find_library_path/single_file_suffix"
```

Comment:
This PR has been overtaken by some others that might help now: We should use the new `BATS_*_TMPDIR` variables for this.